src/assets/styles/collections/_projects.css

```diff
@@ -120,14 +120,16 @@
             inset 0 -0.1875rem 0 0 var(--fl-fgColor, transparent);
         flex-direction: row;
         height: 39.625rem;
-        margin-block-start: 0;
+        align-items: center;
```

Comment:
You've added a duplicate rule for `align-items` here which is a bit confusing.

Reply:
Hey @greatislander, now that title and the anchor tag are in a div with absolute position on the anchor tag, we don't need align-items here I think.